submissions/natayakovleva/HTML-CSS-popup/index.html

```diff
@@ -0,0 +1,65 @@
+              </label>
+              <div class="popup">
+                <div class="popup-main">
+                  <ul class="popup-main__list">
+                    <li class="popup__item popup__item-calendar"><a class="popup__item-link" href="#">Calendar</a></li>
+                    <li class="popup__item popup__item-mail"><a class="popup__item-link" href="#">Mail</a></li>
+                    <li class="popup__item popup__item-news"><a class="popup__item-link" href="#">News</a></li>
+                    <li class="popup__item popup__item-maps"><a class="popup__item-link" href="#">Map</a></li>
+                    <li class="popup__item popup__item-files"><a class="popup__item-link" href="#">Files</a></li>
+                    <li class="popup__item popup__item-contacts"><a class="popup__item-link" href="#">Contacts</a></li>
+                    <li class="popup__item popup__item-tanslation"><a class="popup__item-link" href="#">Translate</a></li>
+                    <li class="popup__item popup__item-videos"><a class="popup__item-link" href="#">Videos</a></li>
+                    <li class="popup__item popup__item-search "><a class="popup__item-link" href="#">Search</a></li>
+                    <li class="popup__item popup__item-photos"><a class="popup__item-link" href="#">Photo</a></li>
+                    <li class="popup__item popup__item-apps"><a class="popup__item-link" href="#">Apps</a></li>
+                    <li class="popup__item popup__item-documents"><a class="popup__item-link" href="#">Document</a></li>
+                  </ul>  
+                </div>
+                <input class="popup-main__input" type="checkbox" id="popup-main__input">
+                <label class="popup-main__label" for="popup-main__input">
+                  <span class="popup-main__checkbox">More</span>
+                </label>
+                <div class="popup-add">
+                  <ul class="popup-add__list list__reset">
+                    <li class="popup__item popup__item-chat"><a class="popup__item-link" href="#">Chat</a></li>
+                    <li class="popup__item popup__item-classroom"><a class="popup__item-link" href="#">Classroom</a></li>
+                    <li class="popup__item popup__item-collection"><a class="popup__item-link" href="#">Collection</a></li>
+                    <li class="popup__item popup__item-mars"><a class="popup__item-link" href="#">Mars</a></li>
+                    <li class="popup__item popup__item-blogs"><a class="popup__item-link" href="#">Blogs</a></li>
+                    <li class="popup__item popup__item-promo"><a class="popup__item-link" href="#">Promos</a></li>
+                  </ul>
+                  <a href="#" class="popup__link">Even more from Kottans</a>  
+                </div>
+              </div>
+            </div>
+          </li>
+          <li><a href="#" class="header__link"><img class="header__link-img" src="./img/bell.png" alt="Go to notifications page"></a></li>
```

Comment:
One of the points in the provided article is no need to populate `alt` attribute for decorative images. Icon is a decorative image)